packages/shared/lib/core/layer-2/actions/fetchSelectedAccountLayer2Balance.ts

```diff
@@ -45,6 +47,18 @@ export function fetchSelectedAccountLayer2Balance(): void {
 async function getSelectedAccountLayer2BalanceForAddress(
     selectedAccountEvmAddress: string,
     chain: IChain
+): Promise<{ balance: number; tokenId: string }[] | undefined> {
+    const layer2BaseAndIrc30Balances = await getSelectedAccountLayer2NativeTokenBalancesForAddress(
+        selectedAccountEvmAddress,
+        chain
+    )
+    const erc20Balances = await getSelectedAccountLayer2Erc20BalancesForAddress(selectedAccountEvmAddress, chain)
+    return [...(layer2BaseAndIrc30Balances ?? []), ...(erc20Balances ?? [])]
```

Comment:
`layer2BaseAndIrc30Balances` is always at least an `[]` so we dont need the `?? [] `

Reply:
same for `erc20Balances`